modules/aws_mysql/aws-mysql.json

```diff
@@ -29,6 +29,10 @@
       "description": "Add deletion protection to stop accidental db deletions",
       "default": false
     },
+    "identifier": {
+      "type": "string",
+      "description": "Identifier for the RDS instance."
```

Comment:
let's say `The name of the RDS instance`
from https://registry.terraform.io/modules/terraform-aws-modules/rds/aws/latest#input_identifier